src/js/ko-components/templates/orifice-calculator.haml

```diff
@@ -39,3 +39,4 @@
         %div
           %input.compressibility-correction{ :type => "radio", "data-bind" => "checkedValue: $data, checked: $component.chosenCompressibilityCorrection"  }
           %label{ "data-bind" => "text: $data" }
+      %span{ "data-bind" => "visible: operatingPressure() > 200, text: operatingPressureWarningMessage" }
```

Comment:
Instead of having magic numbers on the template, rely on the operatingPressureWarningMessage being defined or not.

`%span{ "data-bind" => "visible: operatingPressureWarningMessage, text: operatingPressureWarningMessage" }`